[2024-12-17T14:32:43.054+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T14:32:43.093+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: instacart_etl.etl_dim_department manual__2024-12-17T14:32:27.126134+00:00 [queued]>
[2024-12-17T14:32:43.112+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: instacart_etl.etl_dim_department manual__2024-12-17T14:32:27.126134+00:00 [queued]>
[2024-12-17T14:32:43.113+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-17T14:32:43.137+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_dim_department> on 2024-12-17 14:32:27.126134+00:00
[2024-12-17T14:32:43.198+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=10981) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-17T14:32:43.201+0000] {standard_task_runner.py:72} INFO - Started process 10993 to run task
[2024-12-17T14:32:43.215+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'instacart_etl', 'etl_dim_department', 'manual__2024-12-17T14:32:27.126134+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/instacart_etl_dag.py', '--cfg-path', '/tmp/tmpqkcefwgn']
[2024-12-17T14:32:43.227+0000] {standard_task_runner.py:105} INFO - Job 58: Subtask etl_dim_department
[2024-12-17T14:32:43.378+0000] {task_command.py:467} INFO - Running <TaskInstance: instacart_etl.etl_dim_department manual__2024-12-17T14:32:27.126134+00:00 [running]> on host ad3472fa3861
[2024-12-17T14:32:43.905+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='instacart_etl' AIRFLOW_CTX_TASK_ID='etl_dim_department' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T14:32:27.126134+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-17T14:32:27.126134+00:00'
[2024-12-17T14:32:43.908+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T14:32:46.945+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-17T14:32:47.255+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T14:32:47.264+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=instacart_etl, task_id=etl_dim_department, run_id=manual__2024-12-17T14:32:27.126134+00:00, execution_date=20241217T143227, start_date=20241217T143243, end_date=20241217T143247
[2024-12-17T14:32:47.632+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T14:32:48.188+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-12-17T14:32:48.332+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-17T14:32:48.343+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
